Cargo.toml

```diff
@@ -110,7 +110,7 @@ aleph-bft = { package = "fedimint-aleph-bft", version = "0.36.0", default-featur
 anyhow = "1.0.98"
 aquamarine = "0.5.0"
 argon2 = "0.5.3"
-arti-client = { package = "fedimint-arti-client", version = "0.20.0", default-features = false }
+arti-client = { version = "0.21.0", default-features = false }
```

Comment:
@oleonardolima do you remember why we forked arti-client? This change "works" to get everything to build but I don't know if we forked for a different reason other than getting sqlite to link correctly.

Reply:
Oh maybe it was because of rustls?

Reply:
IIRC it was in order to have a version of `tor-dirmgr` compatible with `rusqlite` used by `ldk-node`, otherwise it breaks due to libsqlite3-sys versions.